factory-package-news/announcer.py

```diff
@@ -54,9 +54,9 @@
 Please check the known defects of this snapshot before upgrading:
 https://openqa.opensuse.org/tests/overview?distri=opensuse&groupid=1&version=Tumbleweed&build={version}
 
-When you reply to report some issues, make sure to change the subject.
-It is not helpful to keep the release announcement subject in a thread
-while discussing a specific problem.
+Please do not reply to this email to report issues, rather file a bug
+on bugzilla.opensuse.org for more information on filing bugs please
```

Comment:
Feels like there is a full stop missing after 'bugzilla.opensuse.org'

Reply:
I thought about putting a comma, but I tend to not like punctuation after url's especially full stops, they tend to break things in places

Reply:
actually, a full stop would be no problem, as the 'technically correct' name would actually be ```bugzilla.com.``` (the last dot indicating 'DNS root')

Reply:
Done, didn't put one at the end of the paragraph though because that does screw up the wiki.